activity_calendar/urls.py

```diff
@@ -22,7 +22,9 @@
         ])),
     ])),
 
-    path('api/calendar/ical', CESTEventFeed(), name='icalendar'),
+    path('api/calendar/ical', PublicCalendarFeed(), name='icalendar'),
+    path('api/calendar/birthdays/', BirthdayCalendarFeed(), name='ical_birthdays'),
```

Comment:
It should probably be noted in the calendar preferences form that the birthday calendar is public. In the future, we might want to generate a unique link per member.